commands.go

```diff
@@ -431,82 +204,23 @@ type Cmdable interface {
-	PubSubShardNumSub(ctx context.Context, channels ...string) *MapStringIntCmd
-
-	ClusterMyShardID(ctx context.Context) *StringCmd
-	ClusterSlots(ctx context.Context) *ClusterSlotsCmd
-	ClusterShards(ctx context.Context) *ClusterShardsCmd
-	ClusterLinks(ctx context.Context) *ClusterLinksCmd
-	ClusterNodes(ctx context.Context) *StringCmd
-	ClusterMeet(ctx context.Context, host, port string) *StatusCmd
-	ClusterForget(ctx context.Context, nodeID string) *StatusCmd
-	ClusterReplicate(ctx context.Context, nodeID string) *StatusCmd
-	ClusterResetSoft(ctx context.Context) *StatusCmd
-	ClusterResetHard(ctx context.Context) *StatusCmd
-	ClusterInfo(ctx context.Context) *StringCmd
-	ClusterKeySlot(ctx context.Context, key string) *IntCmd
-	ClusterGetKeysInSlot(ctx context.Context, slot int, count int) *StringSliceCmd
-	ClusterCountFailureReports(ctx context.Context, nodeID string) *IntCmd
-	ClusterCountKeysInSlot(ctx context.Context, slot int) *IntCmd
-	ClusterDelSlots(ctx context.Context, slots ...int) *StatusCmd
-	ClusterDelSlotsRange(ctx context.Context, min, max int) *StatusCmd
-	ClusterSaveConfig(ctx context.Context) *StatusCmd
-	ClusterSlaves(ctx context.Context, nodeID string) *StringSliceCmd
-	ClusterFailover(ctx context.Context) *StatusCmd
-	ClusterAddSlots(ctx context.Context, slots ...int) *StatusCmd
-	ClusterAddSlotsRange(ctx context.Context, min, max int) *StatusCmd
-
-	GeoAdd(ctx context.Context, key string, geoLocation ...*GeoLocation) *IntCmd
-	GeoPos(ctx context.Context, key string, members ...string) *GeoPosCmd
-	GeoRadius(ctx context.Context, key string, longitude, latitude float64, query *GeoRadiusQuery) *GeoLocationCmd
-	GeoRadiusStore(ctx context.Context, key string, longitude, latitude float64, query *GeoRadiusQuery) *IntCmd
-	GeoRadiusByMember(ctx context.Context, key, member string, query *GeoRadiusQuery) *GeoLocationCmd
-	GeoRadiusByMemberStore(ctx context.Context, key, member string, query *GeoRadiusQuery) *IntCmd
-	GeoSearch(ctx context.Context, key string, q *GeoSearchQuery) *StringSliceCmd
-	GeoSearchLocation(ctx context.Context, key string, q *GeoSearchLocationQuery) *GeoSearchLocationCmd
-	GeoSearchStore(ctx context.Context, key, store string, q *GeoSearchStoreQuery) *IntCmd
-	GeoDist(ctx context.Context, key string, member1, member2, unit string) *FloatCmd
-	GeoHash(ctx context.Context, key string, members ...string) *StringSliceCmd
-
-	ACLDryRun(ctx context.Context, username string, command ...interface{}) *StringCmd
-	ACLLog(ctx context.Context, count int64) *ACLLogCmd
-	ACLLogReset(ctx context.Context) *StatusCmd
+	MemoryUsage(ctx context.Context, key string, samples ...int) *IntCmd
 
 	ModuleLoadex(ctx context.Context, conf *ModuleLoadexConfig) *StringCmd
 
+	ACLCmdable
+	HashCmdable
+	HyperLogLogCmdable
+	GeoCmdable
+	GenericCmdable
+	ListCmdable
+	SetCmdable
+	SortedSetCmdable
+	ClusterCmdable
+	ScriptingFunctionsCmdable
+	StringCmdable
+	PubSubCmdable
 	GearsCmdable
 	ProbabilisticCmdable
 	TimeseriesCmdable
```

Comment:
Did you guys intend to omit StreamCmdable from this list?

Reply:
This is a significant breaking change without `StreamCmdable` available

Reply:
@chayim Can you verify if this was intentional? If not, I think we need an urgent new point release including it. If it was intentional, it should be added to the release notes with workaround / migration instructions. Thanks in advance!

Reply:
looks like there's a PR in flight to fix this https://github.com/redis/go-redis/pull/2725

Reply:
`BitMapCmdable` is also missing.

PR to fix -> https://github.com/redis/go-redis/pull/2737